src/crimson/os/seastore/transaction_manager.h

```diff
@@ -484,31 +507,20 @@ class TransactionManager : public ExtentCallbackInterface {
       mapping.is_indirect()
 	? mapping.get_intermediate_key()
 	: mapping.get_key();
-    auto intermediate_base =
-      mapping.is_indirect()
-      ? mapping.get_intermediate_base()
-      : mapping.get_key();
 
     LOG_PREFIX(TransactionManager::clone_pin);
     SUBDEBUGT(seastore_tm, "len={}, laddr_hint={}, clone_offset {}",
       t, mapping.get_length(), hint, intermediate_key);
     ceph_assert(is_aligned(hint, epm->get_block_size()));
-    return lba_manager->clone_extent(
+    return lba_manager->clone_mapping(
       t,
       hint,
       mapping.get_length(),
       intermediate_key,
       mapping.get_val(),
-      intermediate_key
-    ).si_then([this, &t, intermediate_base](auto pin) {
-      return inc_ref(t, intermediate_base
-      ).si_then([pin=std::move(pin)](auto) mutable {
-	return std::move(pin);
-      }).handle_error_interruptible(
-	crimson::ct_error::input_output_error::pass_further(),
-	crimson::ct_error::assert_all("not possible")
-      );
-    });
+      intermediate_key,
+      mapping.get_length()
+    );
```

Comment:
Suggest to append a comment line to `TM::clone_pin()`: "the cloned extent must be stable."

Reply:
Done:-)